### PR TITLE
Update value-proposition.md

### PR DESCRIPTION
**Pull request by jleebee**
Final english and conventions revision :)

### CODE REVIEW

**Review by davidsrz — DISMISSED** [review]

File: docs/about/value-proposition.md [hunk 1]
```diff
@@ -2,50 +2,50 @@
 
 Hermez network project has some properties that will provide value to the users and the community:
 
-## Production ready solution
+## Production Ready Solution
 Hermez will launch in mainnet a project which is intended to be a usable token transfers network from day one. This product will have support, maintenance, and technological evolution to provide users with the most updated functionalities with the right risk management.
```
Comment (druiz0992) [inline docs/about/value-proposition.md:7]: Remove blank line

**Review by druiz0992 — CHANGES_REQUESTED** [review]
Small change requested.

Regards

**Review by jleebee — COMMENTED** [review]
Requested edits have been made, thanks!

File: docs/about/value-proposition.md (hunk 1 above)
Comment (druiz0992) [reply docs/about/value-proposition.md:6]: Can you remove extra line between Production Ready solution section and Decentralized?

**Review by druiz0992 — APPROVED** [review]

**Comment by druiz0992** [comment]
DavidS already approved previous version.

**Pull request merged**